src/docs/command-line-deployment.md

```diff
@@ -81,7 +81,17 @@ The script `destroy_terraform.sh` at [src/scripts/destroy_terraform.sh](/src/scr
   1. The Global variables file
   1. The directory that contains the main.tf and *.tfvars variables file of the configuration to apply
 
-For example, from the root of this repository, you could apply Tier 0 with a command like:
+
+This deployment is performed using tiered model to protect systems using a set of buffer zones between full control of the Environment and the high-risk environment that attackers frequently compromise. The deployment must be started with saca-hub followed by tier-0 , tier-1 etc. 
```

Comment:
Can we link to out for the network explanation instead? I assume this will end up in a bigger doc someday too.
```suggestion
The hub network must be deployed first. See [Networking](https://github.com/Azure/missionlz#networking) for a description of the hub and spoke and what each network is used for.
```